.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,12 @@
-Fixes #<issue_number_goes_here>
+For an explanation of how to fill out the fields, please see the relevant section
+in [PULL_REQUESTS.md](https://github.com/envoyproxy/envoy/blob/master/PULL_REQUESTS.md)
 
-> It's a good idea to open an issue first for discussion.
-
-- [ ] Tests pass
-- [ ] Appropriate changes to README are included in PR
\ No newline at end of file
+Commit Message:
+Additional Description:
+Risk Level:
+Testing:
+Docs Changes:
+Release Notes:
+[Optional Runtime guard:]
```

Comment:
Let's remove this and depcreated/release notes/risk level? These are not so relevant here